api/alembic/versions/b557469a7727_modify_unique_constraint_on_planning_.py

```diff
@@ -0,0 +1,25 @@
+"""Modify unique constraint on planning_weather_stations
+
+Revision ID: b557469a7727
+Revises: d99fcdc4800d
+Create Date: 2022-01-17 10:15:33.448252
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b557469a7727'
+down_revision = 'd99fcdc4800d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_unique_constraint('unique_station_code_for_planning_area',
```

Comment:
There's an existing unique constraint on this table just for station code, removing it was missed in this migration and causes some issues locally, hence the `api/alembic/versions/af108f346073_remove_redundant_constraint.py` to get prod into a good state.